### DISCUSSION TITLE
Support for Exasol database as datasource

### DISCUSSION THREAD

**Discussion by KevKrtz** [great-expectations/great_expectations#5168] [Ideas]
**Is your feature request related to a problem? Please describe.**
It is indeed possible to connect a Exasol database via SQLAlchemy and ODBC to GE but creating a new expectations suite leads to an error:

```
(pyodbc.ProgrammingError) ('42000', '[42000] [EXASOL][EXASolution driver]syntax error, unexpected TEMPORARY_, expecting VIRTUAL_SCHEMA_ [line 1, column 8] (Session: 1703191614157357056) (-6811776) (SQLExecDirectW)')
[SQL: CREATE TEMPORARY TABLE "ge_tmp_458747f7" AS select * from dm_sales.dim_product]
(Background on this error at: http://sqlalche.me/e/14/f405)
```

Unfortunately, Exasol does not support temporary tables.

**Describe the solution you'd like**
GE supporting Exasol databases as default database backend would be awesome. Maybe there is another solution for temporary tables? Maybe views or creating "real" tables and dropping them when session ends?

**Describe alternatives you've considered**
1) Dumping to be tested tables as csv files to a S3 bucket but this does not scale.
2) Modifing the library to simply create a "normal" table but a whole lot of modules are affected.

**Additional context**
None

**Comment by talagluck** [comment]
Hi @KevKrtz - thanks for raising this issue, and apologies for delays! We won't be able to prioritize this internally in the immediate future. That said, we always welcome contributions, and I'm happy to offer assistance in the form of PR review, and any other guidance that might be helpful. Is this something in which you might be interested?
